tests/python/pants_test/backend/python/test_python_chroot.py

```diff
@@ -82,6 +82,39 @@ def dumped_chroot(self, targets):
       finally:
         python_chroot.delete()
 
+  def test_shebang_modified(self):
+      self.create_file(relpath='src/python/test/main.py', contents=dedent("""
```

Comment:
4 space indents in this file are killing you under pythonstyle checks in CI. Please fix the indents up so we can get this in!